config/helm/chart/default/templates/Common/csi/daemonset.yaml

```diff
@@ -170,9 +170,7 @@ spec:
           - name: MAX_UNMOUNTED_VOLUME_AGE
```

Comment:
Looks good to me. It works for me, one only has to wait until webhook pods get restarted. Should we document that somewhere? I am thinking of the case that someone applies `kubernetes.yaml`, then immediately after `kubernetes-csi.yaml` and then immediately after the respective dynakube, one still gets this error until the webhook restarted. Is this a realistic problem, such that we need to give the advise, that one has to at least wait for the webhook pods?

Reply:
Apart from my question I would approve.

Another thing. Do you think we run into problems when keeping the env var in lower case?

Reply:
Great question, could you write that on the ticket so PO/PM can think about it, as that will not change the implementation.

But this will not fix the problem of the scripts that are applying `kubernets.yaml` and `...-csi.yaml` in sequence followed by some wait.
It will fix the fact that the user will not have to uninstall the Operator to fix the issue.

Reply:
Yeah sure, will do 👍